packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-12/tools/vat-status.mjs

```diff
@@ -73,15 +74,11 @@ const NonNullish = val => {
 };
 
 /**
- * @param {string[]} argv
- * @param {{ dbOpen: typeof import('better-sqlite3'), HOME?: string }} io
+ * @param {string} vatName
  */
-const main = async (argv, { dbOpen, HOME }) => {
-  const [_node, _script, vatName] = argv;
-  if (!vatName) throw Error('vatName required');
-
+export const main = async (vatName) => {
```

Comment:
now that this function is a named export, please give it a name descrbing what it does.